.github/policies/resourceManagement.yml

```
@@ -0,0 +1,43 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
```

This needs to be `eventResponderTasks` for responding to GitHub events.

```suggestion
     eventResponderTasks:
```